pubsub/cloud-client/pom.xml

```diff
@@ -31,7 +31,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
     <maven.compiler.source>1.8</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <pubsub.version>0.9.4-alpha</pubsub.version>
+    <pubsub.version>0.8.0</pubsub.version>
```

Comment:
Probably because this is marked alpha and we've configured the versions plugins to ignore alpha releases.

Need to update https://github.com/GoogleCloudPlatform/java-repo-tools/blob/master/versions-rules.xml